src/function_run_result.rs

```diff
@@ -10,6 +10,7 @@ pub struct FunctionRunResult {
     pub memory_usage: u64,
     pub logs: String,
     pub output: serde_json::Value,
+    pub error: Option<String>,
```

Comment:
If there's an error, is there ever an output? I wonder if it makes sense to represent the successful and failed runs as separately where successful runs have `output: serde_json::Value` and failed runs have `error: String`. We could probably do this with an enum.